style.css

```diff
@@ -547,3 +547,152 @@ h3 {
   transform: matrix(-0.26, 0.97, 0.97, 0.26, 0, 0);
   opacity: 0.9;
 }
+
+.contact-us {
+  display: flex;
+  flex-direction: column;
+  height: 800px;
+  width: 327px;
+  background: #191765;
+  padding-top: 10px;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.contact-us h2 {
+  text-align: center;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #9390ff;
+}
+
+.contact-us p {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  color: #f5f5ff;
+  height: 35px;
+  padding: 15px;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  align-content: center;
+  width: 327px;
+  padding-top: 40px;
+  padding-right: 24px;
+}
+
+input[type='text'] {
+  width: 327px;
+  height: 48px;
+  align-self: center;
+  padding-left: 10px;
+  margin-top: 40px;
+  margin-bottom: 12px;
+  background: #fff;
+  box-sizing: border-box;
+}
+
+input[type='email'] {
+  width: 327px;
+  height: 48px;
+  align-self: center;
+  padding-left: 10px;
+  margin-top: 12px;
+  background: #f7f3ff;
+  box-sizing: border-box;
+}
+
+textarea {
+  width: 327px;
+  height: 148px;
+  margin-top: 24px;
+  padding-left: 12px;
+  background: #fff;
+  box-sizing: border-box;
+  resize: none;
+}
+
+.email-description {
+  position: absolute;
+  height: 20px;
+  width: 200px;
+  align-self: center;
+  left: 90px;
+  top: 4970px;
+}
+
+.contact-buttton {
+  background-color: #7d73fc;
+  font-style: normal;
+  color: #edecff;
+  text-align: center;
+  text-decoration: none;
+  border: none;
+  font-size: 12px;
+  line-height: 24px;
+  width: 89px;
+  height: 48px;
+  margin: 20px;
+}
+
+.contact-buttton:hover {
+  background: #7d73fc;
+  border: 1px solid #7d73fc;
+  box-shadow: 0 8px 16px rgba(125, 115, 252, 0.5);
+}
+
+.contact-buttton:active {
+  background: #5e42d7;
+  border: 1px solid #7d73fc;
+}
+
+.contact-buttton:disabled {
+  background: #2a2b33;
+}
```

Comment:
- [ ] [REQUIRED] You have done an awesome job implementing the button interaction, however, you need to implement for `enabled`, `hover`, `pressed`, and `disabled`. Please fix this.
![Screenshot from 2022-02-23 00-01-33](https://user-images.githubusercontent.com/58692590/155234575-895f240d-f754-4059-89d0-47ce6ea03c35.png)